chapters/7-publication.tex

```diff
@@ -353,50 +330,48 @@ \subsection{De-identifying data for publication}
+you may not be able to publicly release any data at all.
+You will still be expected to catalog and cite your data,
+even if you cannot release it publicly.
+In practice, this may mean publishing only a catalog entry
+providing information about the contents of the datasets
+and how future users might request permission to access them
+(even if you are not the person to grant that permission).
+In some cases, it may be possible to release the dataset but 
+embargoing specific variables that are required for the analysis but cannot be released publicly.
+It may be necessary to grant access to the embargoed data for specific purposes,
 such as a computational reproducibility check required for publication,
 if done under careful data security protocols and approved by an IRB.
 
 \subsection{Publishing raw and constructed datasets}
 
 % options for when the data cannot be published
-Publicly documenting all original data generated as part of a research project
+Publicly documenting all original data acquired as part of a research project
 is an important contribution in its own right.
 Cataloging and/or archiving original datasets
-is a significant contribution that can be made
-in addition to any publication of analysis results.\sidenote{
+is a significant contribution in addition to any publication of analysis results.\sidenote{
 	\url{https://www.povertyactionlab.org/sites/default/files/resources/J-PAL-guide-to-publishing-research-data.pdf}}
-If you are not able to publish the data itself,
-due to licensing agreements or ethical concerns,
-there are often options to catalog or archive the data.
-These may take the form of metadata catalog entries or embargoed releases.
-Such setups allow you to hold an archival version of your data
-which your publication can reference,
-and provide information about the contents of the datasets
-and how future users might request permission to access them
-(even if you are not the person to grant that permission).
-They can also provide for timed future releases of datasets
-once the need for exclusive access has ended.
-In all cases, you should choose a method that allows you
-to obtain a digital object identifier (DOI)
-and a formal citation for your data to use in your publication.
-
-% platforms for publishing data
 Publicly releasing data allows other researchers
 to validate the mechanical construction of your results,
 investigate what other results might be obtained from the same population,
 and test alternative approaches or answer other questions.
 This fosters collaboration and may enable researchers to explore variables and
 questions that you do not have time to focus on otherwise.
-There are different options for public data release.
+
+% platforms for publishing data
+The first step toward data publication is choosing the platform where you will publish your data.
+A variety of options exist; 
+it is important to choose one that allows you to obtain a digital object identifier (DOI)
```

Comment:
Add definition of DOI. This is the first time we mention it. What is different from this and a URL?

Reply:
add sidenote @bbdaniels

Reply:
@bbdaniels

Reply:
Done in f6eff4990e615c745f54a1e1ae6b42ff1186ec85

Reply:
and d8ffae355a31439c96c91142ff7f0a69e49e500d